### PR TITLE
Fixes #2029 Exclude CDNJS Version of jQuery From Defer

### LINKED ISSUE #2029: Exclude jQuery served from CDNJS from being deferred
When safe mode for jQuery is checked the CDNJS version of jQuery should also be excluded from being deferred. 

cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js

### PR DESCRIPTION
**Pull request by ScottTravisHartley**
This excludes the CDNJS version of jQuery from defer if the jQuery safe JS option is checked.

### CODE REVIEW

**Comment by hellofromtonya** [comment]
@crystinutzaa Merge into 3.4.4 instead of `develop` to ensure it goes through a QA and release cycle.

File: tests/Integration/Functions/Options/TestExcludeDeferJS.php [hunk 1]
```diff
@@ -37,7 +37,8 @@ public function testShouldReturnExcludeDeferJSArrayWhenSafeMode() {
             'widget.reviews.co.uk',
             '/wp-includes/js/jquery/jquery.js',
             'c0.wp.com/c/(?:.+)/wp-includes/js/jquery/jquery.js',
-            'ajax.googleapis.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
+			'ajax.googleapis.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
+			'cdnjs.cloudflare.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
```
Comment (hellofromtonya) [inline tests/Integration/Functions/Options/TestExcludeDeferJS.php]: Align all of the array elements.

File: tests/Unit/Functions/Options/TestExcludeDeferJS.php [hunk 2]
```diff
@@ -56,7 +56,8 @@ public function testShouldReturnExcludeDeferJSArrayWhenSafeMode() {
             'widget.reviews.co.uk',
             '/wp-includes/js/jquery/jquery.js',
             'c0.wp.com/c/(?:.+)/wp-includes/js/jquery/jquery.js',
-            'ajax.googleapis.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
+			'ajax.googleapis.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
+			'cdnjs.cloudflare.com/ajax/libs/jquery/(?:.+)/jquery(?:\.min)?.js',
```
Comment (hellofromtonya) [inline tests/Unit/Functions/Options/TestExcludeDeferJS.php]: Align all of the array elements.

Comment (hellofromtonya) [reply tests/Unit/Functions/Options/TestExcludeDeferJS.php]: Resolved 2efd86a7203e7acba9d23525d8f86364846b81f9

**Comment by crystinutzaa** [comment]
Yes, I think all 3 issues you sent can be merged in v3.4.4.
@arunbasillal what do you think?

**Comment by arunbasillal** [comment]
@crystinutzaa I agree. Let's send this to 3.4.4

**Comment by hellofromtonya** [comment]
Thank you @ScottTravisHartley for contributing to WP Rocket! @crystinutzaa updated our tests to validate your contribution. All looks good and this PR has been approved for merge as part of Rocket 3.4.4.

We appreciate your contribution 👍

**Comment by ScottTravisHartley** [comment]
No problem thanks for getting it approved!

**Pull request merged**